resources/views/partials/front-page-hero.blade.php

```diff
@@ -0,0 +1,10 @@
+<div class="row">
+    <div class="col-md-6">
+        <h2>{!! $get_hero_page->post_title !!}</h2>
```

Comment:
I think we'll want to process titles through `htmlspecialchars` on titles, no? Not certain what benefit we derive leaving it un-escaped with `!!` Same consideration should be given to other content below.